### ISSUE TITLE
Theming: sane default - minimum fg/bg contrast 

### ISSUE THREAD

**Issue by dimaqq** [warpdotdev/warp#249]
### Describe the solution you'd like?

```shell
iTerm2 has a feature that enforces minimum contrast between background and foreground.
I think warp could benefit from such feature too.
```

### Is your feature request related to a problem? Please describe.

```shell
Out of the box, `tig` (pretty git graph) displays "selected line" as white text on lime background in warp.

This makes the test rather hard to read. Maybe it's a problem with `tig`, or with `warp` or with number-of-colours detection(?).

I think there will be other programs where colours are misconfigured, thus a default to ensure bad bg/fg combinations are still legible would be nice.

Environment as seen by programs:

TERM_PROGRAM=WarpTerminal
TERM=xterm-256color
```
```


### Additional context

![Screenshot 2021-10-06 at 12 19 46](https://user-images.githubusercontent.com/662249/136135605-90f7275b-8697-435c-b8c9-e333110912b6.png)

Example of unreadable bg/fg combo, first line.

**Comment by elviskahoro** [comment]
@dimaqq 
I also use tig and had the same issue in warp. Thanks for articulating this!

**Comment by elviskahoro** [comment]
@dimaqq this fix should be coming out tomorrow, btw.

**Comment by dimaqq** [comment]
Confirming that it's fixed in 2021.11.02

**Comment by yvt** [comment]
Is there any way to disable the minimum contrast enforcement? It breaks editor themes with subtle colors and Powerline styling (#861).
